tests/utils.go

```diff
@@ -1340,6 +1340,17 @@ func renderContainerSpec(imgPath string, name string, cmd []string, args []strin
 		Image:   imgPath,
 		Command: cmd,
 		Args:    args,
+		SecurityContext: &k8sv1.SecurityContext{
+			Privileged:               NewBool(false),
```

Comment:
Just a note; if we don't explicitly specify the user id here, we will have to wait for a 0.58.1 release of vm-killer
Which may result in our tests failing until that happens

Reply:
why? I think kubevirtci uses its locally built version.

Reply:
sorry, I meant testing which does not run on kubevirtci